os/fs/driver/mtd/smart.c

```diff
@@ -18,7 +18,7 @@
 /****************************************************************************
  * fs/driver/mtd/smart.c
  *
- * Sector Mapped Allocation for Really Tiny (SMART) Flash block driver.
+ * Journaling Sector Mapped Allocation for Really Tiny (jSmartFs) Flash block driver.
```

Comment:
Abbr. is the SMART (Sector Mapped Allocation for Really Tiny), not SMARTFS. Is it correct to write FS together in the brackets? That's why its file name is smart.c. There are smartfs_xx.c files separately.

same as belows in the file. Is it SMART or SMARTFS?

Reply:
I agree, this part should not be changed.
Or at least not right now without proper consideration of the meaning.

Reply:
This comment still remains pending